### ISSUE TITLE
Chrome Mobile: nextTick is not a function

### ISSUE THREAD

**Issue by jonashaag** [polyfillpolyfill/polyfill-service#1119]
```
TypeError: Property 'nextTick' of object function (n){var t,r=this;if(!o(r)||r._s!==S)throw v(N);if(r._s=$,H&&(r[K]=_()),n!==tn){if(!i(n))throw v(Q);t=a(n)(m(r,U),m(r,O)),t===nn&&k(r,O,t.e)}} is not a function
at ? (/v2/polyfill.min.js:3:9613)
at k(/v2/polyfill.min.js:3:10960)
at T(/v2/polyfill.min.js:3:11175)
at ? (/v2/polyfill.min.js:3:10329)
...
```

Sorry that I don't have a non-minified version. Does polyfill.io provide source maps?

User Agent:

```
Mozilla/5.0 (Linux; Android 4.4.2; de-de; SAMSUNG GT-I9195 Build/KOT49H) AppleWebKit/537.36 (KHTML, like Gecko) Version/1.5 Chrome/28.0.1500.94 Mobile Safari/537.36
```

Not sure how to debug this further... I don't have a GT-I9195 to test this.

**Comment by ruiaraujo** [comment]
It helps if you post the polyfill url that you are using.

**Comment by CheloXL** [comment]
You can use [for example](https://cdn.polyfill.io/v2/polyfill.js?ua=Mozilla/5.0%20(Linux;%20Android%204.4.2;%20de-de;%20SAMSUNG%20GT-I9195%20Build/KOT49H)%20AppleWebKit/537.36%20(KHTML,%20like%20Gecko)%20Version/1.5%20Chrome/28.0.1500.94%20Mobile%20Safari/537.36) get the uncompressed version that is being sent to that browser.

Looking at that, I see that nextTick is being used in the Promise polyfill, but that polyfill is already minimized on the non-minimized version of polyfill, so it will be quite hard to debug the issue anyways...

**Comment by jonashaag** [comment]
This is the weirdest glitch I've found in my entire existence.

If you add a `<div id="process">` to the DOM, it breaks.

The problem is that Yaku checks for `window.process`, which seems to be defined in IE as `$("#process")`. Yaku then tries to use `process.nextTick`, which of course doesn't exist.

**Comment by jonashaag** [comment]
https://github.com/ysmood/yaku/issues/43

**Comment by triblondon** [comment]
Merged upstream.  We need to update to >=0.17.9.

**Comment by JakeChampion** [comment]
There is a PR to update the dependencies in our shrinkwrap which will fix this -- https://github.com/Financial-Times/polyfill-service/pull/1136/files#diff-36018a6a9e1de1798516799b84841318R1893

It is probably best to update `package.json` as well with the version @triblondon has mentioned.

**Comment by JakeChampion** [comment]
Resolved by #1150
